docker-compose.yml

```diff
@@ -44,6 +44,7 @@ services:
       - BB_CLIENT_CERT_FILE=../shared_files/bb-dev-test-cert.pem
       - BB_CLIENT_KEY_FILE=../shared_files/bb-dev-test-key.pem
       - BB_SERVER_LOCATION=https://fhir.backend.bluebutton.hhsdevcloud.us
+      - BB_CHECK_CERT=false
```

Comment:
is there a corresponding ops ticket for this environment variable?

Reply:
Shouldn't need to set it for our AWS environments because we do want to check the BB certificate from those (which is the default)

Reply:
makes sense